docs/TiC2000.md

```diff
@@ -0,0 +1,35 @@
+# Running TI-C2000 compiler
+
+This document is a very basic explanation of how to get the TI-C2000 compiler working with compiler explorer.
+
+# Prerequisites
+
+To run the TI-C2000 compiler you will need a valid installation of the C2000 compiler. https://www.ti.com/tool/C2000-CGT
+
+## Setting up c++.local.properties
+
+The next step is to create a `c++.local.properties` file under `etc/config` folder. The next step is going to be
+different for everyone, as you can choose what compiler options you pass to the compiler and so on, but I'm going to
+paste my template here, and you can just modify, what you need
+
+```
+compilers=&c2000
+# TI-C2000 Compilers
+group.c2000.compilers=c2k2260LTS
+group.c2000.versionFlag=-version
+group.c2000.supportsBinary=false
+group.c2000.intelAsm=
+group.c2000.compilerType=tic2000
+group.c2000.demangler=C:\ti\ccs1210\ccs\tools\compiler\ti-cgt-c2000_22.6.0.LTS\bin\dem2000.exe
+group.c2000.demanglerType=tic2000
+group.c2000.includeFlag=-I
+
+compiler.c2k2260LTS.exe=C:\ti\ccs1210\ccs\tools\compiler\ti-cgt-c2000_22.6.0.LTS\bin\cl2000.exe
+compiler.c2k2260LTS.options=-I"C:\ti\ccs1210\ccs\tools\compiler\ti-cgt-c2000_22.6.0.LTS\include"
+compiler.c2k2260LTS.name=C2000 22.6.0.LTS
+```
+
+## Running Compiler Explorer
```

Comment:
This is hopefully covered in the main `README.md`. These instructions seem very specific to Windows, maybe?